File: pe-lwjgl-binding/src/main/java/org/saar/lwjgl/opengl/fbo/attachment/IAttachment.java
package org.saar.lwjgl.opengl.fbo.attachment;

import org.saar.lwjgl.opengl.fbo.attachment.index.AttachmentIndex;

public interface IAttachment {

    void init(AttachmentIndex index);

    void allocate(int width, int height);

    void delete();

}

File: pe-lwjgl-binding/src/main/java/org/saar/lwjgl/opengl/fbo/attachment/IAttachment.kt
package org.saar.lwjgl.opengl.fbo.attachment

import org.saar.lwjgl.opengl.fbo.attachment.index.AttachmentIndex

interface IAttachment {

    fun attach(fbo: Int, index: AttachmentIndex)

    fun allocate(width: Int, height: Int)

    fun delete()

}
